dojo/filters.py

```diff
@@ -1456,9 +1457,9 @@ class ApiFindingFilter(DojoFilter):
     jira_change = DateRangeFilter(field_name="jira_issue__jira_change")
     last_reviewed = DateRangeFilter()
     mitigated = DateRangeFilter()
-    mitigated_on = DateFilter(field_name="mitigated", lookup_expr="exact")
-    mitigated_before = DateFilter(field_name="mitigated", lookup_expr="lt")
-    mitigated_after = DateFilter(field_name="mitigated", lookup_expr="gt")
+    mitigated_on = DateTimeFilter(field_name="mitigated", lookup_expr="exact")
+    mitigated_before = DateTimeFilter(field_name="mitigated", lookup_expr="lt")
+    mitigated_after = DateTimeFilter(field_name="mitigated", lookup_expr="gt")
```

Comment:
If a user were to be reliant on passing a date to this filter, would the API accept the absence of the timestamp? For example, if the query was `mitigated_on=2025-01-02` would a validation error be thrown?

Reply:
A validation error does not get thrown if you use 2025-01-02 without the timestamp. Should it?

Reply:
No, a validation error should not be raised. I was asking to make sure that this change would not cause any issues for users that were currently using dates